rules/TypeDeclaration/Rector/Param/ParamTypeFromStrictTypedPropertyRector.php

```diff
@@ -103,6 +103,10 @@ private function decorateParamWithType(ClassMethod $classMethod, Param $param):
             return null;
         }
 
+        if ($classMethod->isAbstract()) {
```

Comment:
before this PR a few things above the loop were resolved even though not necessary as a method without a body is not worth introspecting